### PR TITLE
Add missing includes to Geometry/Tracker(Numbering/Geometry)Builder

### PR DESCRIPTION
**Pull request by davidlange6**
as usual - found in module build errors

### CODE REVIEW

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: Geometry/TrackerGeometryBuilder/BuildFile.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
 <use name="Geometry/CommonDetUnit"/>
 <use name="Geometry/CommonTopologies"/>
 <use name="Geometry/TrackerNumberingBuilder"/>
+<use name="CondFormats/GeometryObjects"/>
```
Comment (smuzaffar) [inline Geometry/TrackerGeometryBuilder/BuildFile.xml:10]: @davidlange6 , looks like this is causing cyclic deps in cxxmodules IBs due to following dep chain
CondFormats/GeometryObjects -> Geometry/VeryForwardGeometryBuilder -> Geometry/TrackerNumberingBuilder -> CondFormats/GeometryObjects

module build failed to compile any thing,

**Comment by davidlange6** [comment]
Indeed. - however this PR not causing cyclic dependencies. Its admitting to having them..

#31420 is causing them.

On Sep 4, 2020, at 8:36 AM, Malik Shahzad Muzaffar <notifications@github.com<mailto:notifications@github.com>> wrote:



@smuzaffar commented on this pull request.

________________________________

In Geometry/TrackerGeometryBuilder/BuildFile.xml<https://github.com/cms-sw/cmssw/pull/31356#discussion_r483416626>:

> @@ -7,6 +7,10 @@
 <use name="Geometry/CommonDetUnit"/>
 <use name="Geometry/CommonTopologies"/>
 <use name="Geometry/TrackerNumberingBuilder"/>
+<use name="CondFormats/GeometryObjects"/>


@davidlange6<https://github.com/davidlange6> , looks like this is causing cyclic deps in cxxmodules IBs due to following dep chain

CondFormats/GeometryObjects -> Geometry/VeryForwardGeometryBuilder -> Geometry/TrackerNumberingBuilder -> CondFormats/GeometryObjects


module build failed to compile any thing,

**Comment by smuzaffar** [comment]
correct but if this PR along with https://github.com/cms-sw/cmssw/pull/31343/files gets merged then it will break all IBs :-)
By the way, all the cyclic deps can be found under ignominy results https://cmssdt.cern.ch/SDT/cgi-bin/newQA.py?arch=slc7_amd64_gcc820&release=CMSSW_11_2_X_2020-09-03-2300#ignominy

**Comment by davidlange6** [comment]
On Sep 4, 2020, at 8:44 AM, Malik Shahzad Muzaffar <notifications@github.com<mailto:notifications@github.com>> wrote:



correct but if this PR along with https://github.com/cms-sw/cmssw/pull/31343/files gets merged then it will break all IBs :-)

Yes, its time to revert #31420.

By the way, all the cyclic deps can be found under ignominy results https://cmssdt.cern.ch/SDT/cgi-bin/newQA.py?arch=slc7_amd64_gcc820&release=CMSSW_11_2_X_2020-09-03-2300#ignominy

Right - Did you not notice that I fixed 4 of them yesterday?..

**Comment by davidlange6** [comment]
please test with #31359

**Comment by davidlange6** [comment]
my local full build shows that these two changes work and restore the modules build to a working state (mostly)

**Comment by cvuosalo** [comment]
+1

**Comment by davidlange6** [comment]
this pr does not require any external change..

**Comment by silviodonato** [comment]
+1

**Pull request merged**